### ISSUE TITLE
[TIL] Day10 [오세광] 200911

### ISSUE THREAD

**Issue by sekwangoh** [codestates/ds-TIL#214]
# 키워드: 스프린트 챌린지, 블로깅

# (Act)What you actually did.
1. 오전 3시간 동안 스프린트 챌린지 
2. 오후 블로깅

# (Observe)What can you observe at your action’s outcome
1. 스프린트 챌린지 
- 사소한 실수(문법, [] 등)으로 시간 많이 잡아 먹는다. 
- 특정 조건을 충족하는 데이터 프레임을 만드는게 익숙하지 않다. 
- 특정 수에 top5 같은 등수를 내는게 익숙지 않다.

2. 블로깅
- 스스로가 공부한 내용을 글을 쓰는 것이 어렵다. 

# (Evaluate)How you evaluate your action's outcome vs. the initial goal(plan).

1. 주말 동안 과제를 쭉 훑어 나가면서, 사소한 문법, 특정상황에서 쓰는 키워드, 내용들을 정리한다. 
2. 특히 특정조건과 특정 수를 구하는 문제들을 조금 더 파본다. 
3. 쫓기지말고 마음 편히 내가 배운 내용들을 글로 쓰며 즐거웠던 마음을 다시 찾아보자

**Comment by jhan8823** [comment]
df.loc에 익숙해지시면 뚝딱뚝딱 만드실 수 있습니다 :) 1주일간 고생하셨습니다

**Comment by sekwangoh** [comment]
> df.loc에 익숙해지시면 뚝딱뚝딱 만드실 수 있습니다 :) 1주일간 고생하셨습니다

네! 주말동안 내용 정리해서, 뚝딱뚝딱 만들 수 있게 손과 머리가 무의식으로 움직이도록 해보려고요 ㅎㅎ 1주일동안 고생이 많으셨어요. 정민님 :)
